test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -57,23 +51,23 @@ public void isAnyNull_multipleObjectArgsShouldReturnFalse() {
     
     @Test
     public void elementsAreUnique_emptySetShouldReturnTrue() {
-        Set<Object> set = new HashSet<Object>();
+        ArrayList<String> set = new ArrayList<String>();
         assertTrue("empty set must return true", Utils.elementsAreUnique(set));
     }
     
     @Test
     public void elementsAreUnique_singleElementSetShouldReturnTrue() {
-        Set<Object> set = new HashSet<Object>();
-        set.add(new Object());
+        ArrayList<String> set = new ArrayList<String>();
```

Comment:
Since you are creating a set for every test, you might want to look at doing it at setup.